### DISCUSSION TITLE
[FFmpeg-devel] Implement support for Vulkan multiplane images and video decoding

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#297] [General]
Looks like FFMPEG master will be getting native Vulkan decode soon.  What are the ramification for the Vulkan support that UG currently has?  Or is FFMPEG Vulkan for decode, and UG Vulkan is for display?  Is there overlap?

http://ffmpeg.org/pipermail/ffmpeg-devel/2023-February/306661.html

**Comment by MartinPulec** [comment]
> Is there overlap?

I believe that it would be possible to implement passing decoded Vulkan surface to be displayed directly by the Vulkan display so that copying into main memory could be avoided.
